fdbserver/DataDistribution.actor.cpp

```diff
@@ -4122,9 +4156,13 @@ ACTOR Future<Void> ddSnapCreate(DistributorSnapRequest snapReq, Reference<AsyncV
 		if (e.code() != error_code_operation_cancelled) {
 			snapReq.reply.sendError(e);
 		} else {
+			// enable DD should always succeed
+			ASSERT(setDDEnabled(true, snapReq.snapUID));
```

Comment:
```suggestion
			bool success = setDDEnabled(true, snapReq.snapUID);
			ASSERT(success);
```
`setDDEnabled` has side-effects, I would prefer having `ASSERT`-statements be side-effect free (If we ever want to compile them out in production.... Also someone unfamiliar with FDB-code might be confused by that as most projects would remove assertions in production builds).